main/uwsgi/APKBUILD

```diff
@@ -9,14 +9,18 @@ pkgdesc="uWSGI application container server"
 url="http://projects.unbit.it/uwsgi/"
 arch="all"
 license="GPL2"
-depends="mailcap"
+depends="
+        mailcap
+        jansson
```

Comment:
You are right, its not needed:

````
>>> uwsgi*: Tracing dependencies...
        mailcap
        busybox
        so:libc.musl-x86_64.so.1
        so:libcap.so.2
        so:libcrypto.so.38
        so:libjansson.so.4
        so:libpcre.so.1
        so:libssl.so.39
        so:libxml2.so.2
        so:libz.so.1
````

note the automatic dependency of `so:libjansson.so.4`